Source/WebCore/loader/cache/CachedScript.h

```diff
@@ -47,8 +47,6 @@ class CachedScript final : public CachedResource {
 private:
     bool mayTryReplaceEncodedData() const final { return true; }
 
-    bool shouldIgnoreHTTPStatusCodeErrors() const final;
```

Comment:
Does this function still need to exist in the base class? Does it still need to be a virtual function in the base class?